### PR TITLE
fix noise from jacoco

### PR DESCRIPTION
**Pull request by rymurr**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/1105)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1093,6 +1093,7 @@ limitations under the License.
                 <configuration>
                   <dataFileIncludes>
                     <dataFileInclude>**/jacoco.exec</dataFileInclude>
+                    <dataFileInclude>**/jacoco-quarkus.exec</dataFileInclude>
```
Comment (snazy) [inline pom.xml:1096]: The Quarkus extension just writes `jacoco.exec`, do we need this one?

Comment (rymurr) [reply pom.xml:1096]: https://quarkus.io/guides/tests-with-coverage#running-the-tests-with-coverage it actually doesn't. We can change it to write `jacoco.exec` if this test works though, and remove this line.

**Comment by rymurr** [comment]
Increased coverage and reduced the logs by 5000 lines (80%)

File: pom.xml (hunk 1 above)
Comment (snazy) [reply pom.xml]: Ah, right.

**Review by snazy — APPROVED** [review]

**Pull request merged**